frontend/src/actions/LoginActions.js

```diff
@@ -6,6 +6,7 @@ export const sendLogin = (urlPath, requireAuth, body) => {
     return (dispatch, getstate) => {
         HTTPFetchUtil.sendPostRequest(urlPath, requireAuth, body)
             .then((httpResponse) => {
+                
```

Comment:
U're gonna cause conflicts to everyones PR because of this extra line you added >:(